src/UIComponents/Predict.jsx

```diff
@@ -23,13 +24,15 @@ class Predict extends Component {
     setTestData: PropTypes.func.isRequired,
     predictedLabel: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
     confidence: PropTypes.number,
-    getPredictAvailable: PropTypes.bool
+    getPredictAvailable: PropTypes.bool,
+    setPrediction: PropTypes.func
   };
 
   handleChange = (event, feature) => {
     const testData = this.props.testData;
     testData[feature] = event.target.value;
     this.props.setTestData(testData);
+    this.props.setPrediction({});
```

Comment:
One alternative approach would be to have the `SET_TEST_DATA` reducer also clear the `prediction`.  It might be a bit more robust by centralising the business logic in the reducer rather than out here in the UI.

Reply:
Thank you for the feedback! I just pushed an update.